Fix compilation with --disable-filesystem and --enable-xrc

The above configuration didn't compile due to wx/arrstr.h being included
only conditionally via wx/filesys.h

Signed-Off-By: Anders Larsen <al@alarsen.net>
---
 include/wx/xrc/xmlres.h |    1 +
 1 files changed, 1 insertion(+)

Index: b/include/wx/xrc/xmlres.h
===================================================================
--- a/include/wx/xrc/xmlres.h
+++ b/include/wx/xrc/xmlres.h
@@ -17,6 +17,7 @@
 
 #include "wx/string.h"
 #include "wx/dynarray.h"
+#include "wx/arrstr.h"
 #include "wx/datetime.h"
 #include "wx/list.h"
 #include "wx/gdicmn.h"
